app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,14 @@ def show
     @post = Post.find(params[:id])
```

Comment:
- [ ] Good job by creating all the create methods, but it would be better to add a functionality in case the create method didn't work you suppose to add an else statement.
![image](https://user-images.githubusercontent.com/74744787/215283130-11ac2dc1-5820-41a9-b4c9-d41087f3ea1d.png)
![image](https://user-images.githubusercontent.com/74744787/215283142-090581f7-a483-45b0-a5bd-c81d5d1b49e8.png)
![image](https://user-images.githubusercontent.com/74744787/215283157-ecac04ae-ff8a-4fdb-a2c3-734597e243f3.png)
it should look like this 
![image](https://user-images.githubusercontent.com/74744787/215283201-a2972abd-9ea2-415a-b71d-c4f14e3732ed.png)

Reply:
I have implemented the changes.